egress/egress-only-tracking.yaml

```diff
@@ -0,0 +1,84 @@
+components:
+  schemas:
+    EgressOnlyTracking:
+      description: |-
+        One specific egress tracking specification for a specified port. 
+        An application which supports a single or limited number of egress tracking specifications for a single port 
+        should return an error if the number of egress tracking specifications for a specific port or in total exceeds its capabilities.
+      type: object
+      required: [port_name]
+      properties:
+        port_name:
+          description: Name of the received port this egress tracking rule/specification has to be applied.
+          type: string
+          x-constraint:
+          - '/components/schemas/Port/properties/name'
+          x-field-uid: 1
+        metric_tags:
+          description: |-
+            One or more metric tags can be used to enable tracking portion of or all bits
+            in a corresponding header field for metrics per each applicable value.
+            These would appear as tagged metrics in corresponding egress_only_tracking metrics.
+          type: array
+          items: 
+            $ref: '#/components/schemas/EgressOnlyTracking.MetricTag'
+          x-field-uid: 2
+        filter:
+          description: |-
+            Specified a rule which will be used to first filter received packets on this port 
+            before applying the egress tracking metric_tags on them. 
+            If the packet does not match the rule, it will not be considered for egress tracking.
+          $ref: '#/components/schemas/EgressOnlyTracking.Filter'
+          x-field-uid: 3
+    EgressOnlyTracking.MetricTag:
+      description: |-
+        Metric Tag can be used to enable tracking portion of or all bits
+        in a corresponding header field for metrics per each applicable value.
+        These would appear as tagged metrics in corresponding flow metrics.
```

Comment:
The result is not part of corresponding flow metrics. It will be shown in new "egress_only_tracking" metrics

Reply:
done.